src/components/v2/shared/DistributionSplitsSection/DistributionSplitCard.tsx

```diff
@@ -0,0 +1,200 @@
+            <Col span={17}>
+              <div
+                style={{
+                  display: 'flex',
+                  alignItems: 'center',
+                  justifyContent: 'space-between',
+                }}
+              >
+                <span style={{ cursor: 'pointer' }}>
+                  <FormattedAddress address={split.beneficiary} />
+                </span>
+                {isOwner && <CrownFilled />}
+              </div>
+            </Col>
+          </Row>
+        )}
+
+        {parseInt(split.projectId ?? '0') > 0 ? (
+          <Row>
+            <Col span={7}>
+              <label>Beneficiary:</label>
+            </Col>
+            <Col span={17}>
+              <span style={{ cursor: 'pointer' }}>
+                <FormattedAddress address={split.beneficiary} />
+              </span>
+            </Col>
+          </Row>
+        ) : null}
+
+        <Row gutter={gutter} style={{ width: '100%' }} align="middle">
+          <Col span={7}>
+            <label>Percentage / Amount:</label>
+          </Col>
+          <Col span={17}>
+            <div
+              style={{
+                display: 'flex',
+                width: '100%',
+                justifyContent: 'space-between',
+                alignItems: 'center',
+              }}
+            >
+              <span
+                style={{
+                  marginRight: 10,
+                  width: 100,
+                  maxWidth: 100,
+                }}
+              >
+                <Space size="large">
+                  <span>{permyriadToPercent(split.percent)}%</span>
+                  {/* {!targetIsInfinite && (
+                    <span>
+                      <CurrencySymbol currency={currencyName} />
+                      {formatWad(
+                        amountSubFee(parseWad(target), feePerbicent)
+                          ?.mul(split.percent)
+                          .div(10000),
+                        { precision: 4, padEnd: true },
+                      )}
+                    </span>
+                  )} */}
+                </Space>
+              </span>
+            </div>
+          </Col>
+        </Row>
+
+        {isLocked && (
+          <Row gutter={gutter} style={{ width: '100%' }} align="middle">
+            <Col span={7}>
+              <label>Locked: </label>
+            </Col>
+            <Col span={17}>
+              until {formatDate((split.lockedUntil ?? 0) * 1000, 'yyyy-MM-DD')}
+            </Col>
+          </Row>
+        )}
+      </Space>
+
+      {isLocked ? (
+        <LockOutlined style={{ color: colors.icon.disabled }} />
+      ) : (
+        <Button
+          type="text"
+          onClick={e => {
+            onSplitsChanged([
+              ...splits.slice(0, splitIndex),
+              ...splits.slice(splitIndex + 1),
+            ])
+            e.stopPropagation()
+          }}
+          icon={<CloseCircleOutlined />}
+        />
+      )}
+      <DistributionSplitModal
```

Comment:
Instance of `DistributionSplitModal` in each card, rather than a single one from the parent `index`